src/realm.js

```diff
@@ -614,4 +621,25 @@ export class Realm {
     }
     for (; i < generatorBody.length; i++) realmGeneratorBody.push(generatorBody[i]);
   }
+
+  handleError(error: IntrospectionThrowCompletion): boolean {
```

Comment:
We very likely will want to also handle errors that are not Introspection errors.